ungleich/dynamicweb
# Task/3709/faq tos cms template

To test this PR:

 1. As a superuser, create a page in Django CMS for faq/tos
 2. Goto page settings
 3. Fill up the basic and advanced settings as desired
 4. While filling the advanced settings, make sure to change the template to "Data Center Light" and save the settings
 5. Publish the page
 6. Preview the page having logged in as the superuser
 7. You should see the django CMS toolbar. Choose the structure view
 8. Fill in the title and text as desired for both languages
 9. Publish changes
 10. Visit the faq tos url
 11. If everything is ok, we need to create a link to this faq page somewhere
